returnn/tf/network.py

```diff
@@ -1474,13 +1485,13 @@ def _resolve_layer(layer_):
       if layer in end_points:
         continue
       layer_queue.append(layer)
+
```

Comment:
Why did you add this empty line?

Reply:
"Restored" the wrong line, coffee not working yet :)